apps/frontend/src/utilities/export_util.ts

```diff
@@ -1,6 +1,5 @@
-import concat from 'concat-stream';
+import ZipFile from 'adm-zip';
```

Comment:
why did we need to change zip libraries?

Reply:
Because the previous packages did not provide unzipping in memory.